packages/augur-sdk/src/state/getter/Users.ts

```diff
@@ -689,6 +693,26 @@ export class Users {
       }
     );
 
+    const endTime = await augur.contracts.augur.getTimestamp_();
```

Comment:
Instead of doing a call here you could just use Dexie.maxKey in the query for the endTime since the goal is to get everything in that direction

Reply:
endTime is used to calculate start time. Is there a dexie way to saying Dexie.maxKey minus X seconds?

Reply:
ah right, nevermind then. I thought there was somewhere we cached block time and only updated it on new blocks but not really a huge deal as its one call.